openpgp/packet/private_key.go

```diff
@@ -220,7 +225,7 @@ func (pk *PrivateKey) parse(r io.Reader) (err error) {
 		}
 		count := uint32(uint32(n[0])<<24 | uint32(n[1])<<16 | uint32(n[2])<<8 | uint32(n[3]))
 		if !pk.Encrypted {
-			count = count+2 /* two octet checksum */
+			count = count + 2 /* two octet checksum */
```

Comment:
There's a few minor irrelevant changes in this PR, could you revert those?

Reply:
I could, but this are changes automatically introduced by the go linter, so at some point I think we should just commit those